docs/HowTo/Introduction.md

```diff
@@ -12,17 +12,17 @@ tools is to help identify possible problem areas in an assembly and help direct
 The first goal of these tools are to help identify APIs that are not portable among the various .NET Platforms. These 
 include Microsoft supported platforms (Windows, Windows apps, DNX) as well as other implementations, such as Mono and 
 Xamarin.  Some APIs may be removed on certain platforms (such as AppDomains, File I/O, etc), or refactored into other
-types (such as some Reflection APIs). Sometimes, the fix is relatively simple, sometimes not as simple. This tool provides
-information to help guide a developer to rework or rewrite certain parts of an assembly to be more portable and resilient
-to version changes. For details, please see [here](PlatformPortability.md).
+types (such as some Reflection APIs). Sometimes the fix will be relatively simple while other times it may be more involved. 
+This tool provides information to help guide a developer to rework or rewrite certain parts of an assembly to be more portable
+ and resilient to version changes. For details please see [here](PlatformPortability.md).
 
 ## Breaking Changes between .NET 4.x versions
 
-Another goal of the tool is to provide guidance and insight into possible breaking changes on the .NET Framework that may
+Another goal of the tools is to provide guidance and insight into possible breaking changes on the .NET Framework that may
```

Comment:
Preeti had a suggestion to use the singular since ApiPort is a single tool. I think that would be clearer. What do you think?